ikarus/classifier.py

```diff
@@ -113,8 +113,8 @@ def propagate_labels(
         ] = True
         final_pred_proba.loc[certainty_info[f"certain{i}"] == False] = 0.000001
 
-        final_step_mtx = np.dot(connectivities, final_pred_proba.values)
-        final_step_mtx = np.divide(final_step_mtx, final_step_mtx.sum(axis=1))
+        final_step_mtx = connectivities.dot(final_pred_proba.values)
+        final_step_mtx = np.divide(final_step_mtx, final_step_mtx.sum(axis=0))
```

Comment:
The `axis=1` was intended. It should be `final_step_mtx.sum(axis=1, keepdims=True)`, though, to go with your proposal.

Reply:
Oh sorry I didn't see the notification for this. I will correct it now